.github/ISSUE_TEMPLATE/new-cfe-template.yml

```diff
@@ -1,11 +1,29 @@
-name: 🚧 Code freeze exception request
+name: 🚧 Code freeze exception request [RETIRED]
 description: Request to process a particular pull request as a code freeze exception.
 title: "[CFE]: "
 labels: ["code freeze exception"]
 body:
   - type: markdown
     attributes:
       value: |
+        🚧 The code freeze exception approval process is retired. If you need to apply a FIX to the frozen release do one of the following: 🚧
+        
+        **Make a PR against trunk**
+        1. Create your pull request against trunk.
+        2. Add the milestone corresponding to the release you are fixing.  E.g. use milestone `10.0.0` for targeting the `release/10.0` branch.
+        3. Get your pull request reviewed and merged.
+        4. A new pull request will be generated against the release branch with your changes. 
+        
+        **Make a PR against the release branch**
+        1. Create your pull request against the frozen release branch.
+        2. Add the label `cherry pick to trunk` to the PR.
+        3. Get your pull request reviewed and merged.
+        4. A new pull request will be generated against trunk with your changes.
+        
+        It is important that the newly generated pull request is reviewed, tested, and merged as soon as possible to avoid 
+        delaying the next planned build release.
+        
+        ---
```

Comment:
Do we have a preference for one of the two options? I think there's a mention of it in the docs in this same PR. Would it make sense to link here to it or give some more direction about which way to go?